Ui fixes

makenziemoore opened the pull request:
card type icon fixes

mark-gpn committed:
initial commit

bertoncobb committed:
Adding a closing script tag fixes IE 8

mark-gpn committed:
Merge pull request #1 from SecureSubmit/fixing_ie_8_bug

Adding a closing script tag fixes IE 8

bertoncobb committed:
Updating HpsException to fix PHP 5.2 bug

mark-gpn committed:
Merge pull request #2 from SecureSubmit/5.2_exception_bug

Updating HpsException to fix PHP 5.2 bug

Commit:
upgrade module to opencart 2.0.0

The changes reflect the opencart 2.0.0 standards.  There was an issue in
sucuresubmit.tpl that would not allow for '$this->config->get' so I was
forced to use 'global $config' and '$config->get' to allow the system to
fetch the test or public key.  All other development is correct.

Commit:
final iteration of securesubmit module for opencart 2.0

Everything works as expected. You do need to have a SMTP connection(as I
found out the hard way) for the redirect to happen after form completion.

securesubmit-buildmaster committed:
Merge pull request #4 from cmtracy/opencart_2.0_module

Opencart 2.0 module

slogsdon committed:
HpsChargeService -> HpsCreditService. fix deprecation warning

slogsdon committed:
Merge branch 'v1.5.6'

Conflicts:
	catalog/controller/payment/securesubmit.php

slogsdon committed:
Update secure.submit-1.0.2.js

slogsdon committed:
Update HpsService.php

slogsdon committed:
Update HpsConfiguration.php

russelleverett committed:
Merge pull request #5 from hps/update-gateway-domains

update gateway domains

russelleverett committed:
Revert "update gateway domains"

markhagan committed:
Merge pull request #6 from hps/revert-5-update-gateway-domains

Revert "update gateway domains"

ChessMess committed:
Update HpsConfiguration.php

ChessMess committed:
Update HpsTokenService.php

ChessMess committed:
Update secure.submit-1.0.2.js

ChessMess committed:
Update HpsService.php

slogsdon committed:
Merge pull request #7 from hps/update-cert-url

Update cert url to support PCI 3.1

ChessMess committed:
Update HpsService.php

ChessMess committed:
Update HpsTokenService.php

ChessMess committed:
Update secure.submit-1.0.2.js

slogsdon committed:
Merge pull request #8 from hps/update-prod-url

Update production URLs to PCI 3.1

makenziemoore committed:
style updates

*matching styles to woo commerce / magento
*need icons to match card types

makenziemoore committed:
style updates

*matching styles to woo commerce / magento
*need icons to match card types

Commit:
Added velocity checks and updated to latest php-SDK

slogsdon committed:
apply updates and cleanup

remove default files
migrate to latest JS file
attach JS event handlers

slogsdon committed:
apply updates and cleanup

remove default files
migrate to latest JS file
attach JS event handlers

Commit:
tab cleanup

slogsdon committed:
Merge pull request #9 from hps/velocity-checks

Added velocity checks and updated to latest php-SDK

makenziemoore committed:
fix input borders and style confirm order button

makenziemoore committed:
fix input borders and style confirm order button

makenziemoore committed:
Reskin Completed

-removed card owner field
-added card icons
-added grayed out cc

makenziemoore committed:
Reskin Completed

-removed card owner field
-added card icons
-added grayed out cc

slogsdon committed:
remove unnecessary files

ChessMess committed:
Merge pull request #10 from hps/opencart-UI-updates

Opencart ui updates

makenziemoore committed:
changing key order / color icons / update heartland logo

slogsdon committed:
remove non-plugin files

russelleverett committed:
Merge pull request #11 from hps/opencart-UI-updates-rebase

Opencart ui updates rebase

Commit:
Open cart 2.3.0.2 introduced significant structural changes that break previous plugins. This branch applies new requirements defined here http://forum.opencart.com/viewtopic.php?f=183&t=165958

Commit:
code related changesto bring inline with this version of OC

Commit:
replaced code around getting public key for the javascript and fixed directory references in code

Commit:
updated link references

Commit:
updated link references

Commit:
Readme Changes

makenziemoore committed:
blah

Commit:
Initial commit

Commit:
Readme Update

makenziemoore committed:
ui-updates

makenziemoore committed:
missing-image

Commit:
rebase

slogsdon reviewed (changes requested):
@makenziemoore @ttnnkkrr Let's bring in the necessary changes as a new commit. The history from the main repo doesn't need to be in this one, and the rebase has introduced merge conflict artifacts into some of the files.

ttnnkkrr committed:
Update securesubmit.php

Force 2 digit precision and initialize $this->session->data[$HPS_KEY]['lastErrorTime']

ttnnkkrr commented:
Yea i thought i had picked out all the merge conflicts
@makenziemoore I can take care of shanes request if you like

ttnnkkrr commented:
Closing and redoing without history
